### PR TITLE
Fix the Obtain officially supported entry and can view log after failure

### PR DESCRIPTION
**Pull request by zsl1549**
1、To solve the problem ：Obtain officially supported entry and can view log after failure
2、solution ：ClusterID field can be used to view user logs after failure to create QR code
Encapsulated cluster creation progress popup
3、test results : Obtain officially supported entry and can view log after failure
image
image

### CODE REVIEW

File: src/pages/AddCluster/KClusterList/index.js [hunk 1]
```diff
@@ -364,13 +368,19 @@ export default class EnterpriseClusters extends PureComponent {
...
             updateKubernetes={updateKubernetes}
-            updateKubernetesClusterID={clusterID}
+            updateKubernetesClusterID={ID}
           />
           {showBuyClusterConfig && this.renderCreateClusterShow()}
           <Col style={{ textAlign: 'center', marginTop: '32px' }} span={24}>
             <Button onClick={this.preStep}>上一步</Button>
             {nextDisable ? (
-              <Tooltip title="请选择需要初始化的集群">
+              <Tooltip
```
Comment (barnettZQG) [inline src/pages/AddCluster/KClusterList/index.js:377]: 不满足要求的集群是不能选中的效果，还有其他因素也会是不能初始化，看看其他的不能选中及提示。

File: src/components/Cluster/ClusterProgressQuery/index.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            eid={eid}
+            clusterID={clusterID}
+            selectProvider={selectProvider || providerName}
+            onCancel={() => {
+              this.setState({ showCreateLog: false });
+            }}
+          />
+        )}
+        <Row loading={loading} className={styles.box}>
+          <Alert
```
Comment (barnettZQG) [inline src/components/Cluster/ClusterProgressQuery/index.js:66]: 控制企业版不显示钉钉群入口。

Comment (zsl1549) [reply src/components/Cluster/ClusterProgressQuery/index.js:66]: 已修复

File: src/pages/AddCluster/KClusterList/index.js (hunk 1 above)
Comment (zsl1549) [reply src/pages/AddCluster/KClusterList/index.js:377]: 已修复

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**